roles/ara_api/tasks/install/distribution.yaml

```diff
@@ -28,6 +28,15 @@
     name: "{{ ara_distribution_packages }}"
     state: present
 
+# TODO: consider putting the name of the package into a variable if distributions
```

Comment:
Debian uses the same name for this package, so it might not be required to make it a variable :-)